internal/app/generator/generator_test.go

```diff
@@ -0,0 +1,21 @@
+package generator
+
+import "testing"
+
+func BenchmarkUniqGenerate(b *testing.B) {
+	b.Run("generate 10 length", func(b *testing.B) {
```

Comment:
Не стоит использовать пробелы в названиях тестов, так как Go их заменяет автоматически на `_` и это затрудняет поиск теста по имени. Лучше сразу использовать `_` или просто не использовать пробел. 
```suggestion
	b.Run("generate_10_length", func(b *testing.B) {
```